configure.ac

```diff
@@ -762,6 +762,14 @@ if test x$enable_rust != xno; then
   RUST_LIBS="-lrustzcash"
 fi
 
+# Gitian uses a config.site that sets depends_prefix, and then sets --prefix=/
+# build.sh just uses --prefix
+if test x$depends_prefix != x; then
+  LIBSNARK_DEPINST="$depends_prefix"
+else
+  LIBSNARK_DEPINST="$prefix"
+fi
```

Comment:
I think this works [and is POSIX-compliant](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02):
```
LIBSNARK_DEPINST="${depends_prefix:-$prefix}}"
```

Reply:
ACK but I will leave it as-is to be consistent with the rest of the file.